app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
```

Comment:
Fix the errors described below:
2 row: 24 symbol: ANN001 Missing type annotation for function argument 'comfort_class'
2:39: ANN001 Missing type annotation for function argument 'clean_mark'
2:51: ANN001 Missing type annotation for function argument 'brand'
2:57: ANN204 Missing return type annotation for special method
7:1: E302 expected 2 blank lines, found 1
8:24: ANN001 Missing type annotation for function argument 'distance_from_city_center'
8:51: ANN001 Missing type annotation for function argument 'clean_power'
8:64: ANN001 Missing type annotation for function argument 'average_rating'
8:80: ANN001 Missing type annotation for function argument 'count_of_ratings'
8:80: E501 line too long 97 > 79 characters
8:97: ANN204 Missing return type annotation for special method
14:26: ANN001 Missing type annotation for function argument 'cars'
14:31: ANN201 Missing return type annotation for public function
22:39: ANN001 Missing type annotation for function argument 'car'
22:43: ANN201 Missing return type annotation for public function
23:74: W504 line break after binary operator
27:31: ANN001 Missing type annotation for function argument 'car'
27:35: ANN201 Missing return type annotation for public function
31:28: ANN001 Missing type annotation for function argument 'rating'
31:35: ANN201 Missing return type annotation for public function
33:80: E501 line too long 96 > 79 characters)
37:1: E303 too many blank lines (3)
37:48: Q000 Single quotes found but double quotes preferred
38:49: Q000 Single quotes found but double quotes preferred
40:22: W291 trailing whitespace
54:1: W293 blank line contains whitespace